### ISSUE TITLE
Weapon Finesse should be unspecific

### ISSUE THREAD

**Issue by anatoliy-savchak** [GrognardsFromHell/TemplePlus#780]
To be fair, it's only now (after many years) I figured out that Weapon Finesse is taken not on specific weapon, but on any finesse ready weapon.

The feat indeed had specific weapon in mind in 3.0, but it was changed to be unspecific in 3.5. See pics below.

![image](https://github.com/GrognardsFromHell/TemplePlus/assets/40269424/dfdb1ebe-0183-4aac-933e-803146398ded)

I'd say we should allow users to use general unspecific feat as well.

**Comment by doug1234** [comment]
This would be great. Maybe this could be a stricter rules change.

**Comment by dolio** [comment]
Weapon Finesse in ToEE already works the 3.5 way, though.

**Comment by doug1234** [comment]
You are right on that... there is code for the old way but it works the new way.

**Comment by JohnChristianD** [comment]
I don't exactly get the exact problem here.
